package.json

```diff
@@ -58,6 +58,7 @@
     "redux-react-firebase"
   ],
   "dependencies": {
+    "@babel/polyfill": "^7.0.0",
```

Comment:
Maybe that this addition was needed due to node version or `babel-preset-env` settings needing to be changed since it hasn't been needed until now? Not sure.
Either way though, we can leave it and we can investigate more before it goes to `latest`.